### PR TITLE
syllabus (proposal): code review culture

### PR DESCRIPTION
**Pull request by MondeAnna**
## Description:

Greetings all.

<br />

This is a follow up to a discussion with Ryan about alternate ways that recruits could be taught to review each other.

<br />

The primary lessons being:
1. How to decide which part of a review is important
2. How to be tactful during a critique of another's work

<br />

## Reviewers

Stage one has the recruit in the role of the reviewer:

<br />

[The Graceful Reviewer.](https://www.youtube.com/watch?v=XY6eA2_2hOg) Teaches tact and decernment.
[Seed the Culture You Desire.](https://www.youtube.com/watch?v=xT3oFHBDEZY) Discusses the long term benefits of being a graceful reviewer.

<br />

## PR Author

<br />

Stage two has the recruit in the roles of both author and reviewer: [The emphasis is on the learning and teaching aspects of code reviews.](https://www.youtube.com/watch?v=PJjmw9TRB7s)

<br />

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/topics/code-reviews/part-1-reviewers/_index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+prerequisites:
+  hard:
+  - why-and-how-to-review-code
+tags:
+- soft skills
+- communication
+title: Reviewer Communication During Code Reviews
+---
+
+The people who's code you review, are people. Remember this. Treat them with respect.
```
Comment (sheenarbw) [inline content/topics/code-reviews/part-1-reviewers/_index.md]: Three is an extra ,

`The people who's code you review, are people` should be `The people who's code you review are people`

File: content/topics/code-reviews/part-1-reviewers/_index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- Lesson 1: [The Graceful Reviewer](https://www.youtube.com/watch?v=XY6eA2_2hOg)
```
Comment (sheenarbw) [inline content/topics/code-reviews/part-1-reviewers/_index.md]: `The graceful reviewer` is better than Title Case. Title Case is even bad for titles. Just treat english as sentences

File: content/topics/code-reviews/part-2-author/_index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- soft skills
+- communication
+title: How to Ask for a Code Review
+---
+
+You know how to ask for help with code. Code reviews require a similar tact.
+
+
+- [Both Author and Review Pull Requests](https://www.youtube.com/watch?v=PJjmw9TRB7s)
```
Comment (sheenarbw) [inline content/topics/code-reviews/part-2-author/_index.md]: Same thing with normal english versus title case

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Some punctuation and capitalisation best practices.

**Comment by MondeAnna** [comment]
Afternoon Sheena

Now that the capitalisation is mentioned, I can see it everywhere.

**Review by ry-oc — APPROVED** [review]
Changes done

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**